compiler/one-cmds/one-build

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+''''export SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"                  # '''
+''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
+''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
+''''echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command." # '''
+
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import configparser
+import os
+import subprocess
+import sys
+
+import utils as _utils
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to run ONE drivers in customized order')
+
+    _utils._add_default_arg(parser)
+
+    return parser
+
+
+def _parse_arg(parser):
+    args = parser.parse_args()
+    # print version
+    if args.version:
+        _utils._print_version_and_exit(__file__)
+
+    return args
+
+
+def _verify_arg(parser, args):
+    """verify given arguments"""
+    # check if required arguments is given
+    if not _utils._is_valid_attr(args, 'config'):
+        parser.error('-C/--config argument is required')
+
+
+def _get_driver_name(driver_name):
+    return {
+        'one-import-bcq': 'one-import-bcq',
+        'one-import-tf': 'one-import-tf',
+        'one-import-tflite': 'one-import-tflite',
+        'one-optimize': 'one-optimize',
+        'one-pack': 'one-pack',
+        'one-codegen': 'one-codegen'
+    }[driver_name]
+
+
+def _parse_cfg(args):
+    config = configparser.ConfigParser()
+    parsed = config.read(os.path.expanduser(getattr(args, 'config')))
+    if not parsed:
+        raise AssertionError('not found given configuration file')
```

Comment:
Does the message correctly describe the situation? I mean, let's assume there exists a configuration file, but the file has wrong contents and thus cannot be parsed. Is it checked here? If so, the message should be changed to something like 'Configuration file cannot be parsed.'

(optional) How about capitalizing the first character? 'Not found ~~.'

(optional) How about using `SystemExit` or `RuntimeError` instead of `AssertionError` ? Assertion is usually used for debugging purpose.
(https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement)

Reply:
@jinevening `configparser.read` returns the list of configuration file that was parsed successfully. Being `parsed successfully` means here that the file exists. And wrong contents will be caught by `configparser` library. BTW, I'll change the error to `FileNotFoundError`. Thank you for your suggestion.